src/components/NavBar/NavMenu.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import AccountCircle from '@mui/icons-material/AccountCircle';
+import IconButton from '@mui/material/IconButton';
+import Menu from '@mui/material/Menu';
+import MenuItem from '@mui/material/MenuItem';
+import { useSession } from '@inrupt/solid-ui-react';
+
+const NavMenu = ({ menuId, openMenu, setOpenMenu, anchorEl, setAnchorEl }) => {
+  const { session } = useSession();
+  const handleMenuClose = () => {
+    console.log('Closed!');
+    setOpenMenu(false);
+    setAnchorEl(null);
+  };
+
+  return (
+    <Menu
+      anchorEl={anchorEl}
+      anchorOrigin={{
+        vertical: 'top',
+        horizontal: 'right'
+      }}
+      id={menuId}
+      keepMounted
+      transformOrigin={{
+        vertical: 'top',
+        horizontal: 'right'
+      }}
+      open={openMenu}
+      onClose={handleMenuClose}
+      onClick={handleMenuClose}
+      sx={{ marginTop: '40px' }}
+    >
+      <MenuItem>
```

Comment:
The icon and Profile is off-center. I think we can do a simple update for that styling.

![Screen Shot 2023-05-09 at 14 35 18](https://github.com/codeforpdx/PASS/assets/14917816/32f2f722-10c7-4855-932d-e35b04bb1021)

Reply:
I think passing a prop to the `<IconButton>` on line 35 will fix. I believe it is `edge="start"`.

Reply:
I don't see any changes to this in the recent commits, so I'm not resolving this yet.

Find a way to center both the icon and the wording. It looks out of place.

Reply:
I saw the new changes to this. I think it looks okay now.